src/gui/guiutility.cpp

```diff
@@ -23,12 +23,24 @@
 
 #include "common/asserts.h"
 
+namespace {
+    const QStringList whitelistedUrlSchemes = {
+        "http",
+        "https"
+    };
+};
+
 using namespace OCC;
 
 Q_LOGGING_CATEGORY(lcUtility, "nextcloud.gui.utility", QtInfoMsg)
 
 bool Utility::openBrowser(const QUrl &url, QWidget *errorWidgetParent)
```

Comment:
There's exactly one call of that function which doesn't pass nullptr for the second parameter. What about adding a commit before this one in your PR to have nullptr as default value for errorWidgetParent?

This way you could spare the nullptr everywhere else in your PR when openBrowser is called.